homeassistant/components/media_player/squeezebox.py

```diff
@@ -303,3 +304,62 @@ def turn_on(self):
         """Turn the media player on."""
         self._lms.query(self._id, 'power', '1')
         self.update_ha_state()
+
+    def play_media(self, media_type, media_id, **kwargs):
+        """
+        Send the play_media command to the media player.
+
+        If ATTR_MEDIA_ENQUEUE is True, add `media_id` to the current playlist.
+        """
+        if kwargs.get(ATTR_MEDIA_ENQUEUE):
+            self.add_uri_to_playlist(media_id)
+        else:
+            self.play_uri(media_id)
```

Comment:
This may be a coding style conflict. I am never a fan of overloading functions or methods, yes I can collapse them down but I was hoping to update the fade time as a property in a future change which would complicate the if statement more. From a python execution point is the separate function causing a decrease in performance? I am not deeply engaged with the python platform and do not know the full performance characteristics but typically having additional calls like this are non-impactful on performance. 

Let me know where coding style and preferences are for this project so I have better visibility in the future. I have updated with the underscore (_) to indicate they are private.